README.md

```diff
@@ -87,20 +87,19 @@ This will allow you to view rails output in real-time.  You can also add 'byebug
 
 If you run into errors related to the version of bundler when building the `web` container, that likely means you need to pull a newer copy of the base Ruby image specified in `Dockerfile`, e.g., `docker pull ruby:{MAJOR}.{MINOR}-stretch`.
 
-Also, if you run into webpacker related issues, you may need to manually install yarn and compile webpacker in your Docker container (or local laptop if you running that way):
+Also, if you run into webpacker related issues (e.g. [`Node Sass could not find a binding for your current environment`](https://github.com/sul-dlss/argo/pull/1678)), you may need to manually install yarn and compile webpacker from within your Docker container (or natively from your laptop, if that's how you're running the web service, instead of using Docker):
```

Comment:
For docker is this done at build time or run time (i.e., shell into container)?

Reply:
my recollection is that the below commands would use the container to output artifacts to the host filesystem.  which is to say, not shelling into the running container, just using a transient container to output assets.  but my recollection is fuzzy at this point.

sorry for the late response, replying unintentionally but slowly got pushed way down my TODO list.  but @jcoyne also prodded me a couple times recently to follow up.

Reply:
also plausible that the above line and the below two commands can get removed -- might've been made moot by https://github.com/sul-dlss/argo/pull/1686 (see also https://github.com/sul-dlss/argo/pull/1678#discussion_r329572955 and https://github.com/sul-dlss/argo/pull/1678#issuecomment-536619578).  i'd advocate slightly for leaving it since peter and i both found it helpful at one point, but i don't feel strongly.